tests/modules/test_math.py

```diff
@@ -0,0 +1,316 @@
+        'test_math_log_frozenset_class',
+        'test_math_log_frozenset_complex',
+        'test_math_log_frozenset_dict',
+        'test_math_log_frozenset_float',
+        'test_math_log_frozenset_frozenset',
+        'test_math_log_frozenset_int',
+        'test_math_log_frozenset_list',
+        'test_math_log_frozenset_None',
+        'test_math_log_frozenset_NotImplemented',
+        'test_math_log_frozenset_range',
+        'test_math_log_frozenset_set',
+        'test_math_log_frozenset_slice',
+        'test_math_log_frozenset_str',
+        'test_math_log_frozenset_tuple',
+        'test_math_log_int',
+        'test_math_log_int_int',
+        'test_math_log_int_frozenset',
+        'test_math_log_list',
+        'test_math_log_list_frozenset',
+        'test_math_log_None',
+        'test_math_log_None_frozenset',
+        'test_math_log_NotImplemented',
+        'test_math_log_NotImplemented_frozenset',
+        'test_math_log_range',
+        'test_math_log_range_frozenset',
+        'test_math_log_set',
+        'test_math_log_set_frozenset',
+        'test_math_log_slice',
+        'test_math_log_slice_frozenset',
+        'test_math_log_str',
+        'test_math_log_str_frozenset',
+        'test_math_log_tuple',
+        'test_math_log_tuple_frozenset',
+
+        'test_math_pow_bool_frozenset',
+        'test_math_pow_bytearray_frozenset',
+        'test_math_pow_bytes_frozenset',
+        'test_math_pow_class_frozenset',
+        'test_math_pow_complex_frozenset',
+        'test_math_pow_dict_frozenset',
+        'test_math_pow_float_frozenset',
+        'test_math_pow_frozenset_bool',
+        'test_math_pow_frozenset_bytearray',
+        'test_math_pow_frozenset_bytes',
+        'test_math_pow_frozenset_class',
+        'test_math_pow_frozenset_complex',
+        'test_math_pow_frozenset_dict',
+        'test_math_pow_frozenset_float',
+        'test_math_pow_frozenset_frozenset',
+        'test_math_pow_frozenset_int',
+        'test_math_pow_frozenset_list',
+        'test_math_pow_frozenset_None',
+        'test_math_pow_frozenset_NotImplemented',
+        'test_math_pow_frozenset_range',
+        'test_math_pow_frozenset_set',
+        'test_math_pow_frozenset_slice',
+        'test_math_pow_frozenset_str',
+        'test_math_pow_frozenset_tuple',
+        'test_math_pow_int_frozenset',
+        'test_math_pow_int_int',
+        'test_math_pow_list_frozenset',
+        'test_math_pow_None_frozenset',
+        'test_math_pow_NotImplemented_frozenset',
+        'test_math_pow_range_frozenset',
+        'test_math_pow_set_frozenset',
+        'test_math_pow_slice_frozenset',
+        'test_math_pow_str_frozenset',
+        'test_math_pow_tuple_frozenset',
+
+        'test_math_sin_bytearray',
+        'test_math_sin_bytes',
+        'test_math_sin_class',
+        'test_math_sin_complex',
+        'test_math_sin_dict',
+        'test_math_sin_float', # only fails for formatting reasons
```

Comment:
If this is a predictable formatting failure is predictable, you can put the float normalisation into the list [here](https://github.com/pybee/batavia/blob/master/tests/utils.py#L797) - this list compensates for known formatting discrepancies (substituting `value` for `key` in that dictionary at a text-content level)